pkg/cover/store_test.go

```diff
@@ -130,3 +130,22 @@ func TestFileStoreRemove(t *testing.T) {
 	err = store.Remove("http")
 	assert.Error(t, err, fmt.Errorf("no service found"))
 }
+
+// verify issue fix https://github.com/golang/go/issues/56552
+func TestConcurrentRemoval(t *testing.T) {
```

Comment:
it's better to add more direct verification as [this](https://stackoverflow.com/questions/31595791/how-to-test-panics)

Reply:
Hi @CarlJi according to the document we can use `recover()` to catch panic, but our issue is `Fatal Error` which cannot recover. From my understanding `Fatal Error` shouldn't exists in a healthy program. So i'm not sure do we still need this case? Thanks!

Reply:
Hi @CarlJi I think our issue it `Fatal Error` which is not recoverable according to[this](https://stackoverflow.com/a/39289246). 
But I just realize the logic for `fileStore.Set` will inevitably causing data inconsistency as it separate 2 storage sync in 2 lockers. as like this case, even I wait for all the goroutine finished, the serviceMap might still have data not been cleared. 
I'll check it further tmr!